### ISSUE TITLE
Twitter redirects do not work in chrome (Possibly, certainly doesn't work on Ungoogled Chromium)

### ISSUE THREAD

**Issue by MapAnon** [libredirect/libredirect#301]
I know this is a browser that isn't formally supported by libredirect, but every other feature seems to be working just fine, even on Librewolf. I don't know if it's something with Ungoogled Chromium or chromium based browsers in general, but twitter redirects only disable twitter's functionality by making it fail to load anything (Allows me to visit a page, but no content). It does not redirect at all.

Using a fresh install of libredirect 2.0.0. Currently using ungoogled chromium version 100.0.4896.88

**Comment by EdwardLangdon** [comment]
@MapAnon It might show a Blank or static different page, If you are logged into Twitter. It's because of the Twitter cookies which loads before Libredirect could take action. Twitter Redirects works except the Twitter Homepage - https://twitter.com/home due to a recent change where we allowed Tweetdeck to work which was Broken by Libredirect. If you don't understand things. See TLDR;

TLDR; Libredirect will not redirect Twitter Homepage I listed above, But it will do for other things.

@ShiroYasha139 Because of the CSRF policy produced by Twitter cookies we cannot redirect the homepage of Twitter as cookies is loaded before Libredirect can act.

FYI & IIRC, This issue is present in Firefox too.

To know more : 
- https://github.com/libredirect/libredirect/issues/110
- https://github.com/libredirect/libredirect/issues/188#issuecomment-1098113180 (You might try this to fix it)

I am closing this as it's a duplicate. It will be reopened if necessary. Replies and comments are welcome.
